hd/address.go

```diff
@@ -144,7 +131,7 @@ func DerivePublicKeyForPath(pubKeyBytes []byte, chain []byte, path string) []byt
 }
 
 func DerivePrivateKey(privKeyBytes []byte, chain []byte, i uint32, prime bool) ([]byte, []byte) {
-	data := []byte{}
+	data := []byte{} // nolint [ megacheck, deadcode ]
```

Comment:
huh?

Reply:
better to actually fix these than just use `nolint` ... this one can be fixed by using a variable definition instead of the string literal ... `var data []byte`